_docs/quickstart.md

```diff
@@ -124,25 +126,21 @@ $
 ```
 
 To get help on any `flux` subcommand or API program, the `flux help`
-command may be used. For example, to view the man page for the `flux-up(1)`
+command may be used. For example, to view the man page for the `flux-hwloc(1)`
 command, use
 
 ```console
-$ flux help up
+$ flux help hwloc
 ```
 
 ### Interacting with a Flux session
 
 There are several low-level commands of interest to interact with
-a Flux sessions. For example, to view states of broker ranks within
-the current session, `flux up` may be used:
+a Flux sessions. For example, to view system topology from each flux-broker and query the resulting data stored in the Flux Key Value Store, `flux hwloc` may be used:
```

Comment:
Since this is the first mention of the KVS, maybe leave off the mechanism used by `flux hwloc` and just state something like

"For example, to view the total resources available to the current instance, `flux hwloc info` may be used"